docker-compose.yml

```diff
@@ -17,19 +17,18 @@ services:
     environment:
       DEBUG_LEVEL: info
       BEACON_API_PORT: 3500
-      HTTP_ENGINE: ""
-      CORSDOMAIN: "http://lodestar.dappnode"
```

Comment:
@alexpeterson91 do you know how this `CORSDOMAIN` env is used? is it something internal to dappnode or just [cop-pasted from prsym package](https://github.com/dappnode/DAppNodePackage-prysm/blob/7de4e25dae088171599715927d8c45bf9665e6eb/beacon-chain/entrypoint.sh#L49) were it is acutally used